stacks/_templates/opensearch-dashboards.yaml

```diff
@@ -5,6 +5,8 @@ repo:
   url: https://opensearch-project.github.io/helm-charts
 version: 2.14.0
 options:
+  labels:
+    stackable.tech/vendor: Stackable
```

Comment:
I added the vendor label to the opensearch-dashboard service in d956e4d. Otherwise, there are no other possibilities to set additional labels, see [here](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch-dashboards/values.yaml).